demo/classification/higgs/popper/README.md

```diff
@@ -0,0 +1,23 @@
+## Using Popper
+
+[Popper](https://github.com/systemslab/popper) is a tool for defining and executing container-native workflows in Docker, as well as other container engines. This workflow contains a `wf.yml` file that defines a Popper workflow for automatically downloading and verifying data, running the benchmark and generating a report. The steps of the workflow are using the complete [HIGGS data set](https://archive.ics.uci.edu/ml/datasets/HIGGS) from UCI. More details about Popper can be found [here](https://popper.readthedocs.io/).
+
+
+### Instructions:
+
+1. Clone the repository.
+```
+git clone https://github.com/aksnzhy/xlearn.git
+```
+
+2. Install docker.
+
+3. Install the `popper` tool.
+```
+curl -sSfL https://raw.githubusercontent.com/getpopper/popper/master/install.sh | sh
+```
+4. Run the workflow.
+```
+cd xlearn/
+popper run -f demo/classification/higgs/popper/wf.yml
+```
```

Comment:
one other thing I forgot to mention: we can show an example of how to run a single step, and also how to run a step in interactive mode (`popper sh`)